packages/sources/svelte/src/components/selection-controls/VtmnCheckbox/VtmnCheckbox.svelte

```diff
@@ -0,0 +1,42 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+
+  /**
+   * ID of the checkbox
+   * @type {string}
+   */
+  export let id;
+
+  /**
+   * Label of the checkbox
+   * @type {string}
+   */
+  export let label;
+
+  /**
+   * Whether the checkbox is disabled
+   * @type {boolean}
+   */
+  export let disabled;
+
+  let className;
+  /**
+   * @type {string} Custom classes to apply to the component.
+   */
+  export { className as class };
+
+  $: componentClass = cn('vtmn-checkbox', className);
```

Comment:
Like the specs, we can use an 'vtmn-checkbox--indeterminate' [checkbox](https://decathlon.github.io/vitamin-web/@vtmn/showcase-css/?path=/docs/components-selection-controls-checkbox--indeterminate) 

Maybe `export let intermediate` with a boolean ? What do you think ?

Reply:
Done